src/lib/Libwin/accesinfo.c

```diff
@@ -126,7 +132,11 @@ create_secure_dacl(char *user, ACCESS_MASK mask, SID *owner_sid)
 		log_err(-1, __func__, logb);
 		return NULL;
 	}
-	InitializeAcl(ndacl, cbAcl, ACL_REVISION);
+	if (InitializeAcl(ndacl, cbAcl, ACL_REVISION) == 0) {
+		sprintf(logb, "failed to InitializeAcl with errorno %d", GetLastError());
```

Comment:
either say error code, or errno (like linux). errorno is bit unconventional

Reply:
Right, Will change all of them to "errno"